ironfish/src/consensus/verifier.ts

```diff
@@ -409,16 +413,19 @@ export enum VerificationResultReason {
   GRAFFITI = 'Graffiti field is not 32 bytes in length',
   HASH_NOT_MEET_TARGET = 'hash does not meet target',
   INVALID_MINERS_FEE = "Miner's fee is incorrect",
-  INVALID_PREV_HASH = 'invalid previous hash',
+  INVALID_FEE_SUM = 'Fee sum is incorrect',
   INVALID_SPEND = 'Invalid spend',
   INVALID_TARGET = 'Invalid target',
+  INVALID_TRANSACTION_FEE = 'Transaction fee is incorrect',
   INVALID_TRANSACTION_PROOF = 'invalid transaction proof',
   INVALID_PARENT = 'invalid_parent',
   NOTE_COMMITMENT = 'note_commitment',
   NOTE_COMMITMENT_SIZE = 'Note commitment sizes do not match',
   NULLIFIER_COMMITMENT = 'nullifier_commitment',
   NULLIFIER_COMMITMENT_SIZE = 'Nullifier commitment sizes do not match',
   ORPHAN = 'Block is an orphan',
+  PREV_HASH_NULL = 'Previous hash is null',
+  PREV_HASH_UNEQUAL = 'previous hash not equal',
```

Comment:
Maybe this could be `PREV_HASH_INVALID`, and the string could be something like "previousBlockHash does not match expected hash"? Just brainstorming a bit -- I think "not equal" might briefly make readers wonder "not equal to what?"

Reply:
Lets call it PREV_HASH_MISMATCH.